feature-libs/checkout/components/components/delivery-mode/delivery-mode.component.html

```diff
@@ -1,128 +1,65 @@
-<ng-container *cxFeatureLevel="'!4.2.2'">
```

Comment:
Commenting here because it's impossible to comment to files that aren't changed as part of this PR: Please revert the `4.2.2` version in AppModule

Reply:
got it Nik, thanks a lot for your reminder!

Reply:
got it Nik, thanks a lot for your reminder!